src/components/Blogs/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+import SearchBar from "../SearchBar";
+import { Container } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+const NUM_BLOGS_TO_SHOW = 9;
+const BLOGS_URL =
+  "https://public-api.wordpress.com/wp/v2/sites/safeplace102505649.wordpress.com/posts";
+
+export default function Blogs() {
+  const { t } = useTranslation();
+  const [blogPosts, setBlogPosts] = useState([]);
+  const fetchBlogPosts = async () => {
+    const data = await fetch(BLOGS_URL);
+    const fetchedBlogPosts = await data.json();
+    setBlogPosts(fetchedBlogPosts);
+  };
+  useEffect(() => {
+    fetchBlogPosts();
+  }, []);
+  const fetchedBlogPostsArr = blogPosts.slice(0, NUM_BLOGS_TO_SHOW);
```

Comment:
this should be shownBlogPostsArr. blogPosts is actually the fetchedBlogPosts, but you are only showing N of them.

but actually, you shouldn't do it like this. you should use `per_page` in the API. even without knowing the wordpress API it's possible to know that there's probably something like this because usually APIs dont make you load EVERYTHING then cut it in your code, they make you say exactly how much you want to load beforehand (this saves them network)

see per_page

https://developer.wordpress.org/rest-api/reference/posts/#arguments

Reply:
I read this documentation before start with WP API and wasn't so clear to me. 
Thanks for referring it to me again it helped me a lot in cleaning the code. Maybe it's not the cleanest but I am sure it's much better now